ui/jest-playwright.config.js

```diff
@@ -13,5 +13,5 @@ module.exports = {
   launchOptions: {
     headless: true,
   },
-  browsers: ['firefox', 'webkit', 'chromium'],
+  browsers: ['firefox', 'chromium'],
```

Comment:
Matthew and I found an issue when running the UI in Webkit/Safari - it appears that the creation of 2 websockets in close order causes the first of the two to never complete its creation (the onOpen event is never called), meaning that the producer side of the UI never becomes available. Occasionally we did see both websockets happily created, so there is some kind of timing window problem that is only seen on Webkit/Safari.

For this PR, this meant that we saw random failures in the e2e tests, hence temporarily removing webkit from the list of browsers here.